<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QPropCodepage</class>
 <widget class="QDialog" name="QPropCodepage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Codepage</string>
  </property>
  <widget class="QGroupBox" name="IDC_STATIC_CODEPAGE_GROUP">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>14</y>
     <width>482</width>
     <height>186</height>
    </rect>
   </property>
   <property name="title">
    <string>Default Codepage</string>
   </property>
   <widget class="QLabel" name="IDC_STATIC_CODEPAGE">
    <property name="geometry">
     <rect>
      <x>14</x>
      <y>22</y>
      <width>456</width>
      <height>40</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhMultiLine</set>
    </property>
    <property name="text">
     <string>Select the default codepage assumed when loading non-Unicode files:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="IDC_CP_SYSTEM">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>70</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>System codepage</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IDC_CP_UI">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>96</y>
      <width>460</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>According to WinMerge User Interface</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="IDC_CP_CUSTOM">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>122</y>
      <width>240</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom codepage:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="IDC_CUSTOM_CP_NUMBER">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>146</y>
      <width>75</width>
      <height>33</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="IDC_DETECT_CODEPAGE">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>210</y>
     <width>460</width>
     <height>60</height>
    </rect>
   </property>
   <property name="text">
    <string>Detect codepage info for these files: .html, .rc, .xml \nneed to restart session</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="IDC_DETECT_CODEPAGE2">
   <property name="geometry">
    <rect>
     <x>26</x>
     <y>278</y>
     <width>460</width>
     <height>68</height>
    </rect>
   </property>
   <property name="text">
    <string>Detect codepage for text files using mlang.dll\nneed to restart session</string>
   </property>
  </widget>
  <widget class="QComboBox" name="IDC_DETECT_AUTODETECTTYPE">
   <property name="geometry">
    <rect>
     <x>46</x>
     <y>350</y>
     <width>75</width>
     <height>33</height>
    </rect>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContents</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
